libraries/chain/include/eosio/chain/fork_database.hpp

```diff
@@ -67,8 +67,9 @@ namespace eosio::chain {
       /**
        *  Add block state to fork database.
        *  Must link to existing block in fork database or the root.
+       *  @return true if n becomes the new best head.
```

Comment:
```suggestion
       *  @return true if n becomes the new best head (and was not the best head before)
```

Reply:
Maybe update the member of `accepted_block_handle` to be: `is_new_best_head` for clarity throughout.